datasets/gov/vmi/gyventoju_islaidos/vmi.py

```diff
@@ -0,0 +1,187 @@
+#       format_version: '1.3'
+#       jupytext_version: 1.14.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import seaborn as sns
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+# %matplotlib inline
+
+# %% [markdown] tags=[]
+# # IŠLAIDOS
+
+# %% [markdown]
+# Agreguoti duomenys iš juridinių asmenų pateiktos informacijos apie gyventojų patirtas įmokas ir gyventojų deklaruotas išlaidas,
+# kuriomis gali būti mažinamos gyventojų apmokestinamosios pajamos. 
+# https://data.gov.lt/dataset/gyventoju-patirtos-islaidos
+
+# %%
+islaidos=pd.read_csv("data/datasets/gov/vmi/gyventoju_islaidos/GyventojuIslaidos.csv")
+
+# %% [markdown] tags=[]
+# # KLAUSIMAI
+
+# %%
+# Kodėl daug nulinių reikšmių
+# Pataisyti brandos lygiai (visi)
+# Kodėl yra maksimali riba 600000
+
+# %% [markdown] tags=[]
+# # Duomenų apžvalga
+
+# %%
+pd.options.display.max_colwidth = 500
+islaidos.head(5)
+
+# %% [markdown]
+# Kai kurie Dtype neatitinka tipo nurodyto struktūros apraše. Pvz: sav_kodas, 
+
+# %%
+islaidos.info() 
+
+# %%
+islaidos.nunique()
+
+# %% [markdown] tags=[]
+# # Išlaidų rūšys pagal populiarumą
+
+# %% [markdown]
+# Brėžiame diagramą, kuri vaizduoja kiekvienos išlaidų rūšies dažnį šiuose duomenyse.
+
+# %%
+mpl.rc('figure', figsize=(20,10))
+islaidos_dazniai=islaidos.value_counts(['ir_kodas','pavadinimas']).sort_values(ascending = False).to_frame()
+islaidos_dazniai.plot.barh().legend(labels=['Dažnis'])
+
+# %% [markdown]
+# Diagramos matome, kad tarp išlaidų rūšių, kuriomis gali būti mažinamos gyventojų apmokestinamosios pajamos, akivaizdžiai pirmauja gyvybės draudimo įmokos.
+
+# %% [markdown]
+# Taip pat galime pažiūrėt, kaip bendrai yra pasiskirstę visos išlaidų rūšys.
+
+# %%
+vienas = islaidos.set_index(['_id','pavadinimas'])['suma'].unstack() #.plot.box(logy = True)
+fig, ax= plt.subplots(figsize=(20, 10))
+ax.set_xscale('log')
+g = sns.boxplot(data=vienas , palette='rainbow', orient="h", ax=ax)
+plt.tight_layout()
+
+# %% [markdown]
+# Iš boxplot diagramų matome, kad "Gyvybės draudimo įmokos" turi labai daug išsiskiriančių reikšmių. Kyla klausimas dėl 60000 ribos duomenyse. Taip pat matome, kad didžiausią medianą turi "Pastatų, statinių remonto, apdailos (išskyrus renovaciją) darbų išlaidos, o labiausiai svyruojančios išlaidų sumos yra dėl "Būsto kredito, suteikto iki 2008-12-31, palūkanos" ir "Nepilnamečių (iki 18 metų) vaikų (įvaikių, globotinių, kuriems nustatyta nuolatinė globa šeimoje) priežiūros paslaugų išlaidos."
+
+# %%
+rūšis_islaidos = islaidos.loc[islaidos['pavadinimas'] == 'Gyvybės draudimo įmokos.']
+count_row_visos = islaidos.shape[0] 
+count_row_gd = rūšis_islaidos.shape[0]
+def percentage(part, whole):
+ Percentage = 100 * float(part)/float(whole)
+ return str(Percentage)
+percentage(count_row_gd, count_row_visos)
```

Comment:
```suggestion
100 * count_row_gd / count_row_visos
```

Nelabai suprantu kam reikia funkcijos, tokiam paprastam dalykui. Be to, kintamieji rašomi iš mažosios raidės, ir skaičius konvertuoti į float nėra būtina.

Būtų gerai, paskaičiuoti procentą visų rūšių, ne tik vienos ir man atrodo tai labiau tinka aukščiau, prie apžvalgos, kur procento skaičius galėtu būti rodomas prie kiekvieno stulpelio.